specs/market-api.yaml

```diff
@@ -1219,15 +1217,6 @@ components:
             reason:
               $ref: 'common.yaml#/components/schemas/Reason'
 
-    AgreementProposalEvent:
```

Comment:
hmmmm... AgreementProposalEvent is required in CollectDemand response! Otherwise there is no way for Provider to receive message passed when requestor calls ConfirmAgreement().

Reply:
Restored `AgreementEvent` for backward compatibility which is required for Alpha.3